modules/account-lib/src/coin/cspr/transactionBuilder.ts

```diff
@@ -70,26 +108,51 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
+  /**
+   * Set the transaction expirationTime
+   *
+   * @param {string} expirationTime The transaction expirationTime
+   * @returns {TransactionBuilder} This transaction builder
+   */
+  duration(expirationTime: string): this {
```

Comment:
consistency-nit: Rename method
```suggestion
  expiration(expirationTime: string): this {
```